portable/ThirdParty/GCC/RP2040/pico_sdk_import.cmake

```diff
@@ -0,0 +1,62 @@
+# This is a copy of <PICO_SDK_PATH>/external/pico_sdk_import.cmake
```

Comment:
If included, this file should have a copyright statement and SPDX identifier.

Reply:
Is that so? the build checks said it wasn't a requirement. That said i don't mind adding one; it is just a verbatim copy of an intended-use-is-copying-to-your-project file from the SDK which doesn't have a copyright notice